docker/README.md

```diff
@@ -0,0 +1,144 @@
+# FIWARE IoT Agent for the Ultralight 2.0 protocol
+
+[![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
+[![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
+
+An Internet of Things Agent for the Ultralight 2.0 protocol (with
+[AMQP](https://www.amqp.org/), [HTTP](https://www.w3.org/Protocols/) and
+[MQTT](https://mqtt.org/) transports). This IoT Agent is designed to be a bridge
+between Ultralight and the
+[NGSI](https://swagger.lab.fiware.org/?url=https://raw.githubusercontent.com/Fiware/specifications/master/OpenAPI/ngsiv2/ngsiv2-openapi.json)
+interface of a context broker.
+
+It is based on the
+[IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib).
+Further general information about the FIWARE IoT Agents framework, its
+architecture and the common interaction model can be found in the library's
+GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information
+check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
+## How to use this image
+
+The IoT Agent must be instantiated and connected to an instance of the [Orion Context Broker](https://fiware-orion.readthedocs.io/en/latest/), a sample `docker-compose` file can be found below. If the
+`IOTA_REGISTRY_TYPE=mongodb`, a [MongoDB](https://www.mongodb.com/) database instance is also required.
+
+```yml
+version: '3.1'
+
+volumes:
+  mongodb:
+
+services:
+  iot-agent:
+    image: fiware/iotagent-ul
+    hostname: iot-agent
+    container_name: fiware-iot-agent
+    depends_on:
+        - mongodb
+    expose:
+        - "4041"
+        - "7896"
+    ports:
+        - "4041:4041"
+        - "7896:7896"
+    environment:
+        - "IOTA_CB_HOST=orion"
+        - "IOTA_CB_PORT=1026"
+        - "IOTA_NORTH_PORT=4041"
+        - "IOTA_REGISTRY_TYPE=mongodb"
+        - "IOTA_MONGO_HOST=mongo-db"
+        - "IOTA_MONGO_PORT=27017"
+        - "IOTA_MONGO_DB=iotagent-ul"
+        - "IOTA_HTTP_PORT=7896"
+        - "IOTA_PROVIDER_URL=http://iot-agent:4041"
+
+  mongodb:
+    image: mongo:3.6
+    hostname: mongo-db
+    container_name: db-mongo
+    ports:
+        - "27017:27017"
+    command: --bind_ip_all --smallfiles
+    volumes:
+       - mongodb:/data
```

Comment:
A comment should be done about the need of a /data directory in the hosting machine so this example may work. The last sentence could be modified in the following way to cope with it:

```
If the `IOTA_REGISTRY_TYPE=mongodb`, a [MongoDB](https://www.mongodb.com/) database instance is also required. 
The example assumes that you have a `/data` directory in your hosting system in order to hold database files (please 
change if you want to use another one in your case)"
```

Reply:
Fixed - 13e42ee